edenia/antelope-tools
# Refactor/use theme charts links 1347

### Use the Material UI theme to apply styles in the charts and links

### What does this PR do?

- Resolve #1347
- Resolve #1350
- Resolve #1355
- Add a wrapper component with the styles for charts
- Fix accessibility errors

### Steps to test

1. Run the project locally
2. Go to main page
3. Check the links

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e211435c-37b6-4fa1-b719-40b38b4a6a57)
